hi-ml-histopathology/testhisto/testhisto/mocks/base_data_generator.py

```diff
@@ -116,14 +116,22 @@ def generate_mock_histo_data(self) -> None:
         raise NotImplementedError
 
     def mount_pathmnist_dataset(self) -> None:
-        ws = get_workspace()
+        logging.info("get_workspace")
```

Comment:
such progress logging should be human readable, something like "Trying retrieve AML workspace via get_workspace"